packages/augur-sdk/src/state/sync/BulkSyncStrategy.ts

```diff
@@ -20,30 +21,24 @@ export class BulkSyncStrategy extends AbstractSyncStrategy implements SyncStrate
     endBlockNumber: number,
   ): Promise<number> {
     if(!endBlockNumber) throw new Error('Cannot bulk sync forever. Please pass in an endblock');
+    if(startBlockNumber >= endBlockNumber) throw new Error('Starting point of bulk sync must be less than the endpoint')
```

Comment:
Is there a convoluted set of circumstances where these could be *equal*?

Reply:
Perhaps? I suppose with your change below that would be fine